File: Python/6.py
n = 100

def sum_of_squares(n):
    return (n * (n+1) * (2 * n + 1)) / 6

def square_of_sum(n):
    return (n * (n + 1) / 2) ** 2

def difference_of_squares(n):
    print(f"The difference between 'square of sum' and 'sum of squares' = {int(square_of_sum(n) - sum_of_squares(n))}")

difference_of_squares(n)

File: C/6.c
#include "6.h"
#include <stdio.h>

unsigned int sum_of_squares(unsigned int n) {
    return (n * (n+1)*(2 * n + 1)) / 6;;
}

unsigned int square_of_sum(unsigned int n) {
    unsigned int sum = (n * (n + 1) / 2);
    return sum * sum;
}

unsigned int difference_of_squares(unsigned int n) {
    return square_of_sum(n) - sum_of_squares(n);
}

int main(void) {
    unsigned int n = 100;
    unsigned int difference = square_of_sum(n) - sum_of_squares(n);
    printf("The difference between 'square of sum' and 'sum of squares' = %d\n", difference);
    return 0;
}